app/assets/javascripts/discourse/controllers/preferences/account.js.es6

```diff
@@ -243,7 +243,7 @@ export default Ember.Controller.extend(
       },
 
       connectAccount(method) {
-        method.doLogin();
+        method.doLogin(true);
```

Comment:
I am not a fan of unnamed booleans like this. From reading this line of code it's not obvious what `true` does. Please change it to use an object literal like `method.doLogin({ reconnect: true })`